wp-content/themes/theme_name/404.php

```diff
@@ -1,6 +1,10 @@
 <?php
+/**
+ * GSS error page
+ */ 
+
 get_header();
 
-esc_html_e( 'ERROR 404', 'text_domain' ):
+echo __( 'ERROR 404', 'text_domain' );
```

Comment:
We should escape all the translatable strings. In fact we should enforce [escape everything](https://vip.wordpress.com/2014/06/20/the-importance-of-escaping-all-the-things/) paradigm.